src/DynamoCoreWpf/ViewModels/Core/GeometryScalingViewModel.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Dynamo.Core;
+using Dynamo.ViewModels;
+
+namespace ViewModels.Core
+{
+    public class GeometryScalingViewModel : NotificationObject
```

Comment:
comments please

Reply:
Comment added in the next commit: https://github.com/DynamoDS/Dynamo/pull/13656/commits/789f51c4a3c106156940924585e1ca40abd9842b